server/services/forms/schemas/contribution-fulfillment.js

```diff
@@ -1,6 +1,7 @@
 const Form = require('../form-builder');
 const { Component, Choice, ConditionChoice } = require('../component-builder');
 const { getSomuItem } = require('../../../middleware/somu');
+const YEAR_RANGE = 120;
```

Comment:
Might be a bit neater to do this?
```suggestion
const MIN_ALLOWABLE_YEAR = (new Date().getFullYear() - YEAR_RANGE);
```

Reply:
Agreed that makes it more readable, change made